CHANGELOG.md

```diff
@@ -2,10 +2,42 @@
 
 ## Unreleased
 
-- [Pull request #601: Add input classes to implement a part of #428](https://github.com/alphagov/accessible-autocomplete/pull/601)
-- [Pull request #621: Handle query change when the text length does not change](https://github.com/alphagov/accessible-autocomplete/pull/621)
-- [Pull request #620: Add `className` attribute to status component](https://github.com/alphagov/accessible-autocomplete/pull/620)
-- [Pull request #591: Add menuAttributes to fix #361](https://github.com/alphagov/accessible-autocomplete/pull/591)
+### Breaking changes
+
+#### Verify your code does not rely on polyfills we have now removed
+
+The code of the accessible autocomplete is no longer transpilled and polyfilled to support IE8-10, matching [our previous announcement](https://github.com/alphagov/design-system-team-internal/issues/561).
+
+However, because the polyfills create or extend global objects, you might have other code in your service unintentionally relying on the inclusion of these polyfills. You might need to introduce your own polyfills or rewrite your JavaScript to avoid using the polyfilled features.
+
+### New features
+
+#### Options to add classes to parts of the component
+
+Use these new options to add your own classes to parts of the component:
+- `inputClasses` for the `input` element
+- `hintClasses` for the suggestion hint (`input` element appearing when the text typed by the user matches the start of an option)
+- `menuClasses` for the `ul` element listing the options
+
+You can also add custom classes to the `ul` listing the options through the new `menuAttributes` option, providing a `className` or `class` property.
+
+This change was introduced in:
+- [pull request #602: Add `inputClasses` option](https://github.com/alphagov/accessible-autocomplete/pull/602). Thanks to @andreyyudin for contributing this change.
+- [pull request #649: Update options for styling component inputs](https://github.com/alphagov/accessible-autocomplete/pull/649)
+- [pull request #650: Update ways of configuring menu attributes](https://github.com/alphagov/accessible-autocomplete/pull/650)
+
+#### New class on the component's status
+
+Use the new `autocomplete__status` class in your CSS (or `<CSS_NAMESPACE>__status` if you set [the `cssNamespace` option](https://github.com/alphagov/accessible-autocomplete?tab=readme-ov-file#cssnamespace-default-autocomplete)) to customise the styles of the component's status element (hidden element that makes announcements to assistive technologies).
+
+This change was introduced in [pull request #453: Add className attribute to status component](https://github.com/alphagov/accessible-autocomplete/pull/453) (merged via [pull request #620](https://github.com/alphagov/accessible-autocomplete/pull/620)). Thanks to @lennym for contributing this change.
```

Comment:
Felt best to acknowledge the contributor's PR first, to highlight their contribution, and then link to the PR we had to open for merging because we couldn't push on their branch.

Reply:
> Like GitHub does when you reference something in a new issue.

Oh, do you have an example of this, I'm not sure I understand what you mean?

Reply:
https://github.com/alphagov/govuk-frontend/issues/1845#issue-643089233